manifests/kots-helm.yaml

```diff
@@ -175,13 +175,13 @@ spec:
   ## Values from Advanced Configuration
   optionalValues:
     ## BigQuery exports
-    - when: '{{repl ConfigOptionEquals "datawarehousesExportsBigQueryEnabled" "1" }}'
+    - when: '{{repl not (empty (ConfigOption "storageBucketsExportGcp"))}}'
```

Comment:
Just so we are aware, with this approach, we will not be able to test with a **dev license** to leave the exports field undefined since it will be set to the default value so it would never be empty.

Reply:
We can just change to a trial license when required no? 😄